ProceduralForest/main.cpp

```diff
@@ -19,13 +19,161 @@
 #include "Cube.h"
 #include "World.h"
 #include "TreeGenerator.h"
+<<<<<<< Updated upstream
 using namespace glm;
 using namespace std;
+=======
+#include "Camera.h"
+#include "filesystem.h"
+
+
+using namespace glm;
+using namespace std;
+#define SCREEN_WIDTH 1024.0f
+#define SCREEN_HEIGHT 768.0f
+
+unsigned int loadTexture(const char *path);
+unsigned int loadCubemap(vector<std::string> faces);
```

Comment:
neither loadCubeMap or loadTextures should be in main. The only function that should be in main is the display loop and the main function.

Reply:
this is for testing only